From cc8362d4e7c20712d86897522aa9416403437f46 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Tue, 25 Jul 2023 17:24:43 +0200
Subject: [PATCH 1/7] ARM: imx: Drop CONFIG_USE_BOOTCOMMAND=n on i.MX6 DHSOM

This board certainly does use default 'run distro_bootcmd' boot command,
make sure this is set in 'bootcmd' variable.

Upstream-Status: Backport [e47cab0fa134ad6c9a7b50c5569953d2ad296ead]
Fixes: 970bf8603b8 ("Convert CONFIG_USE_BOOTCOMMAND et al to Kconfig")
Signed-off-by: Marek Vasut <marex@denx.de>
Reviewed-by: Tom Rini <trini@konsulko.com>
---
 configs/dh_imx6_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/configs/dh_imx6_defconfig b/configs/dh_imx6_defconfig
index a45d3cb05ff..49b4dfa879c 100644
--- a/configs/dh_imx6_defconfig
+++ b/configs/dh_imx6_defconfig
@@ -37,7 +37,6 @@ CONFIG_FIT_VERBOSE=y
 CONFIG_SPL_FIT=y
 CONFIG_DISTRO_DEFAULTS=y
 CONFIG_BOOTDELAY=3
-# CONFIG_USE_BOOTCOMMAND is not set
 CONFIG_SYS_CONSOLE_OVERWRITE_ROUTINE=y
 CONFIG_SYS_SPL_MALLOC=y
 CONFIG_SPL_SPI_LOAD=y
-- 
2.42.0

